composables/useWeb3Modal.ts

```diff
@@ -4,10 +4,12 @@ const modal = ref()
 
 export default () => {
   const { config } = useWagmi()
+  const { urlPrefix } = usePrefix()
 
   createWeb3Modal({
     wagmiConfig: config,
     projectId: useRuntimeConfig().public.walletConnectProjectId,
+    defaultChain: PREFIX_TO_CHAIN[urlPrefix.value] ?? undefined,
```

Comment:
```suggestion
    defaultChain: PREFIX_TO_CHAIN[urlPrefix.value],
```

It should be the same

Reply:
cc @hassnian

Reply:
> It should be the same

thanks fixed